qiskit/circuit/quantumcircuit.py

```diff
@@ -2840,14 +2840,15 @@ def delay(
         qarg: Optional[QubitSpecifier] = None,
         unit: str = "dt",
     ) -> InstructionSet:
-        """Apply :class:`~qiskit.circuit.Delay`. If qarg is None, applies to all qubits.
+        """Apply :class:`~.~qiskit.circuit.library.Delay`. If qarg is ``None``, applies to all qubits.
```

Comment:
It seems this lives in `qiskit/circuit` 🤔 
```suggestion
        """Apply :class:`~.circuit.Delay`. If qarg is ``None``, applies to all qubits.
```